### PR TITLE
Shivam rai2003 patch 2

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
-_site/
-.sass-cache/
-./Gemfile.lock
+- name: Shivam rai
+#   country: Inia
+#   github: ShivamRai2003
+#   image: Student's image (or representation like a "nick" image)
+#   twitter: Shivam Rai
```
Comment (nikit19) [inline .gitignore:5]: Please don't write anything here

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make this change and make sure that the dimensions of your image is 240 x 240 pixels

**Review by codedsun — CHANGES_REQUESTED** [review]
Image is not of appropriate size.

**Comment by ShivamRai2003** [comment]
now what is the problem

**Review by isuruAb — CHANGES_REQUESTED** [review]
I saw 3 duplicate PRs from you.please check those and close unwanted PRs. please don't add your details to the.gitignore file.

**Pull request closed without merging**